tests/prepare.ts

```diff
@@ -105,6 +111,7 @@ const log = async <T>(label: string, fn: () => Promise<T>) => {
       middleware: 'express',
       middlewareTemplate: './tests/fixtures/custom/custom-tsoa-template.ts.hbs',
       routesDir: './tests/fixtures/custom',
+      routesFileName: 'routes.ts',
```

Comment:
@HoldYourWaffle here's your opportunity to test this feature. If you set this one to `customRoutes.ts` and if the integration tests don't fail, then your feature works.

Reply:
Would changing just this line be enough?